pkgs/dart_mcp/example/workflow_client.dart

```diff
@@ -330,9 +377,9 @@ final class WorkflowClient extends MCPClient with RootsSupport {
   /// Invokes a function and adds the result as context to the chat history.
   Future<void> _handleFunctionCall(gemini.FunctionCall functionCall) async {
     chatHistory.add(gemini.Content.model([functionCall]));
-    logger.stderr(
-      'Calling function ${functionCall.name} with args: '
-      '${functionCall.args}',
+    logSink?.add(
```

Comment:
We can just remove this now - when you pass the log sink all protocol traffic is logged.